concordium-sdk/src/test/testresources/smartcontractschema/lib.rs

```diff
@@ -0,0 +1,242 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! # A Concordium V1 smart contract
+use concordium_std::*;
+use core::fmt::Debug;
+
+/// Your smart contract state.
+#[derive(Serialize, SchemaType)]
+pub struct State {
+    // Your state
+}
+
+/// Your smart contract errors.
+#[derive(Debug, PartialEq, Eq, Reject, Serial, SchemaType)]
+enum Error {
```

Comment:
Maybe this should be called something a bit less generic. E.g. "TestContractError" ?

Reply:
It is never returned from the smart contract. It is just there to make the smart contract valid. Emil said smart contracts must contain errors.

Reply:
You don't have to define a custom type if you don't need it. THere are some predefined errors such as ParseError.

Reply:
I have removed the custom errors and am now using a standard ParseError